edk2toolext/rust_environment.py

```diff
@@ -0,0 +1,442 @@
+# @file rust_environment.py
+#
```

Comment:
Please add an args section to `run`, `get_workspace_toolchain_version` and `verify_workspace_rust_toolchain_is_installed`

e.g.

```
Args:
    custom_tool_checks (Dict[str, RustToolInfo], optional): ...description...
    custom_tool_filters (List[CustomToolFilter], optional): ...description...
```